js_modules/dagit/packages/core/src/ticks/tests/SensorDryRunDialog.mocks.tsx

```diff
@@ -195,19 +164,16 @@ export const PersistCursorValueMock: MockedResponse<SetSensorCursorMutation> = {
   result: {
     data: {
       __typename: 'DagitMutation',
```

Comment:
I found that using `buildDagitMutation` created an extremely huge mock of everything so I had to create the outer `__typename: 'DagitMutation'` by hand.